File: Backend/Shabasher/Shabasher.Core/DTOs/ShabashResponse.cs
using Shabasher.Core.Models;

namespace Shabasher.Core.DTOs
{
    public record ShabashResponse(
        string Id,
        string Name,
        string Description,
        string Address,
        string? AvatarUrl,
        List<ShabashParticipantResponse> Participants,
        DateOnly StartDate,
        TimeOnly StartTime,
        DateTime CreatedAt,
        ShabashStatus Status,
        ShabashRole ActorRole,
        UserStatus ActorStatus
        );
}

File: Frontend/app/src/main/java/com/example/shabasher/data/dto/ShabashResponse.kt
package com.example.shabasher.data.dto

import kotlinx.serialization.Serializable

@Serializable
data class ShabashResponse(
    val id: String,
    val name: String,
    val description: String,
    val address: String,
    val startDate: String,
    val startTime: String,
    val createdAt: String,
    val status: String,
    val participants: List<ShabashParticipantDto> = emptyList()
)